manim/animation/animation.py

```diff
@@ -6,7 +6,7 @@
 
 import typing
 from copy import deepcopy
-from typing import Union
+from typing import Union, Optional, Tuple, Iterator
```

Comment:
I think you should do either `import typing` and add type hints like `typing.*` or import separately. Do not mix both, this will raise an error in pylint.

Reply:
we already had both, I didn't change that.

Reply:
Maybe change now?

Reply:
I'm on it.